src/cookbook/navigation/set-up-universal-links.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Set up universal links for iOS
+description: How to animate a widget from one screen to another
+prev:
+title: Send data to a new screen
+path: /cookbook/navigation/passing-data
+next:
+title: Delete data on the internet
+path: /cookbook/networking/delete-data
+js:
+- defer: true
+  url: https://dartpad.dev/inject_embed.dart.js
+---
+
+<?code-excerpt path-base="cookbook/navigation/deeplink"?>
```

Comment:
Not sure what this line actually does, just copy over from other markdown

Reply:
That expects a matching Dart codebase at the link. The cookbook collection of samples is copied in from https://github.com/flutter/codelabs/tree/main/cookbook